frontend/src/metabase/visualizations/echarts/cartesian/option/series.ts

```diff
@@ -368,7 +368,7 @@ export const buildEChartsStackLabelOptions = (
       const value = datum[seriesModel.dataKey];
 
       if (typeof value !== "number") {
-        return " ";
```

Comment:
Same here, let's keep it as a space

Reply:
I actually created an ECharts patch on a prior PR ([#42985](https://github.com/metabase/metabase/pull/42985)) that was specifically addressing how label are handled by ECharts.
If we keep the `" "` instead of a `""`, it will cause non-empty labels to potentially be hidden because `" "` still has a non-zero length and hence non-zero width and height.